xbmc/music/Album.cpp

```diff
@@ -176,6 +176,28 @@ std::string CAlbum::GetGenreString() const
   return StringUtils::Join(genre, g_advancedSettings.m_musicItemSeparator);
 }
 
+const std::vector<std::string> CAlbum::GetAlbumArtist() const
+{
+  //Get artist names as vector from artist credits
+  std::vector<std::string> albumartists;
+  for (VECARTISTCREDITS::const_iterator artistCredit = artistCredits.begin(); artistCredit != artistCredits.end(); ++artistCredit)
```

Comment:
Then I think we should do it to __all__ the loops of that format in a separate PR for consistency, if you team members think it is a good idea.